apps/contrib/templates/meinberlin_contrib/widgets/dropdown_link.html

```diff
@@ -1,11 +1,12 @@
 <div class="dropdown {% if right %}dropdown--right{% endif %}" {{ attrs }}>
     <button
         type="button"
-        class="dropdown-toggle"
+        class="dropdown-toggle button button--select"
         data-toggle="dropdown"
         aria-haspopup="true"
         aria-expanded="false"
         id="{{ id }}"
+        title="{{ label }}: {{ value_label }}"
```

Comment:
Yes, but it may be truncated. I found that I personally expect a tooltip on truncated buttons.